modules/ob-about-s2i.adoc

```diff
@@ -0,0 +1,10 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="about-s2i_{context}"]
+= source-to-image
+
+You can use the `source-to-image` build strategy with `kaniko` to generate a container file and prepare the source code for image build. You require the `kaniko` strategy to create and push the container image to an output location specified in the `spec.output.image` field of your `Build` CR.
```

Comment:
Recommend putting the why for something first in a sentence, so write to: "Creating and pushing the container image to an output location specified in the in `spec.output.image` field of your `Build` CR requires the kaniko` strategy."